### PR TITLE
chore: ACL filters - Chip group options - Sim's Work

### PR DESCRIPTION
**Pull request by cfullam1**
Sim's work

### CODE REVIEW

File: src/ProofOfConcepts/Chips/ConsumerGroupLabel.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import "@patternfly/react-core/dist/styles/base.css";
```
Comment (riccardo-forina) [inline src/ProofOfConcepts/Chips/ConsumerGroupLabel.js:1]: You should never import PatternFly's css directly, those are already available for you in Storybook.

**Review by riccardo-forina — CHANGES_REQUESTED** [review]
`patternfly.css` and `styles.css` should not be included. Also left a comment about importing PatternFly's CSS, it's something we don't need to (and should not) do.

**Comment by simrandhaliw** [comment]
Thanks for opening up this PR on my behalf, Cliodhna! We can close this one now. I have opened a separate PR with the updated code.

File: src/ProofOfConcepts/Chips/ConsumerGroupLabel.js (hunk 1 above)
Comment (simrandhaliw) [reply src/ProofOfConcepts/Chips/ConsumerGroupLabel.js:1]: Thanks for letting me know! I removed all unnecessary CSS imports in the new PR.

**Pull request closed without merging**